src/components/Select/ComboboxMulti/__tests__/ComboboxMulti.spec.js

```diff
@@ -0,0 +1,181 @@
+    const mockOnBlur = jest.fn();
+
+    const suggestions = SUGGESTIONS;
+
+    let wrapper;
+    let inputNode;
+
+    const setFocusOnInput = () =>
+        wrapper
+            .find('.SelectBase__field')
+            .first()
+            .simulate('click');
+
+    beforeEach(() => {
+        wrapper = mount(
+            <ComboboxMulti
+                suggestions={suggestions}
+                suggestionToString={suggestionToString}
+                inputPlaceholder={inputPlaceholder}
+                clearTitle={clearTitle}
+                noSuggestionsPlaceholder={noSuggestionsPlaceholder}
+                onSelectionChange={mockOnSelectionChange}
+                onInputValueChange={mockOnInputValueChange}
+                onBlur={mockOnBlur}
+                onClearAllSelected={mockOnClearAllSelected}
+            />
+        );
+        inputNode = wrapper.find('input').getDOMNode();
+    });
+
+    describe('rendering', () => {
+        it('should initially render empty component correctly', () => {
+            expect(toJson(wrapper)).toMatchSnapshot();
+        });
+        it('should set focus on the input field', () => {
+            wrapper.setProps({ suggestions: [] });
+            setFocusOnInput();
+            expect(wrapper.find('li')).toHaveLength(0);
+            expect(inputNode).toBe(document.activeElement);
+        });
+        it('should render noSuggestions placeholder when empty suggestions list is passed', () => {
+            wrapper.setProps({ suggestions: [] });
+            wrapper.find('input').simulate('change', { target: { value: 'driver' } });
+            setFocusOnInput();
+
+            expect(toJson(wrapper)).toMatchSnapshot();
+            expect(wrapper.find('li')).toHaveLength(1);
+            expect(
+                wrapper
+                    .find('li')
+                    .childAt(0)
+                    .text()
+            ).toEqual(noSuggestionsPlaceholder);
+        });
+        it('should render all suggestions from the list', () => {
+            setFocusOnInput();
+
+            expect(toJson(wrapper)).toMatchSnapshot();
+            expect(wrapper.find('li')).toHaveLength(suggestions.length);
+        });
+        it('should render selection placeholder when component is not focused', () => {
+            expect(wrapper.find('input').props().placeholder).toEqual(inputPlaceholder);
+        });
+    });
+    describe('focusing and blurring the search field', () => {
+        it('should set focus when wrapper element is clicked', () => {
+            const focusSpy = jest.spyOn(inputNode, 'focus');
+
+            expect(inputNode).not.toBe(document.activeElement);
+            expect(focusSpy).not.toHaveBeenCalled();
+
+            setFocusOnInput();
+
+            expect(inputNode).toBe(document.activeElement);
+            expect(focusSpy).toHaveBeenCalled();
+        });
+        it('should stay focused when suggestion is selected', () => {
```

Comment:
is this really a requirement? It sounds confusing that you click on something and next think you are focused somewhere else, no?

Reply:
I checked it with Carlo, it is a requirement